<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
      <link rel="shortcut icon" href="../../img/favicon.ico" />
    <title>ClusterBLAST - plantiSMASH Documentation</title>
    <link rel="stylesheet" href="../../css/theme.css" />
    <link rel="stylesheet" href="../../css/theme_extra.css" />
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/styles/github.min.css" />
        <link href="../../css/extra.css" rel="stylesheet" />
    
      <script>
        // Current page data
        var mkdocs_page_name = "ClusterBLAST";
        var mkdocs_page_input_path = "modules/clusterblast.md";
        var mkdocs_page_url = null;
      </script>
    
    <!--[if lt IE 9]>
      <script src="../../js/html5shiv.min.js"></script>
    <![endif]-->
      <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/highlight.min.js"></script>
      <script>hljs.highlightAll();</script> 
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
    <div class="wy-side-scroll">
      <div class="wy-side-nav-search">
          <a href="../.." class="icon icon-home"> plantiSMASH Documentation
        </a><div role="search">
  <form id ="rtd-search-form" class="wy-form" action="../../search.html" method="get">
      <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" title="Type search term here" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../..">Home</a>
                </li>
              </ul>
              <p class="caption"><span class="caption-text">Using plantiSMASH</span></p>
              <ul>
                  <li class="toctree-l1"><a class="reference internal" href="../../intro/">Introduction</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../../website_submission/">Submitting jobs on the webserver</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../../understanding_output/overview/">Results page</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../../understanding_output/database/">plantiSMASH database</a>
                  </li>
              </ul>
              <p class="caption"><span class="caption-text">Generic modules</span></p>
              <ul class="current">
                  <li class="toctree-l1"><a class="reference internal" href="../clusterrules/">BGC detection</a>
                  </li>
                  <li class="toctree-l1 current"><a class="reference internal current" href="#">ClusterBLAST</a>
    <ul class="current">
    <li class="toctree-l2"><a class="reference internal" href="#clusterblast-database">ClusterBLAST database</a>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#ranking-system">Ranking system</a>
    </li>
    <li class="toctree-l2"><a class="reference internal" href="#similarity-percentage">Similarity percentage</a>
        <ul>
    <li class="toctree-l3"><a class="reference internal" href="#example-1-low-similarity-good-match">Example 1: low similarity, good match</a>
    </li>
    <li class="toctree-l3"><a class="reference internal" href="#example-2-perfect-similarity-poor-match">Example 2: perfect similarity, poor match</a>
    </li>
    <li class="toctree-l3"><a class="reference internal" href="#example-3-high-similarity-poor-match">Example 3: high similarity, poor match</a>
    </li>
        </ul>
    </li>
    </ul>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../knownclusterblast/">KnownClusterBLAST</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../coexpress/">Coexpress</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../subgroup/">Subgroup Finder</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../tfbs/">TFBS Finder</a>
                  </li>
              </ul>
              <p class="caption"><span class="caption-text">Specific modules</span></p>
              <ul>
                  <li class="toctree-l1"><a class="reference internal" href="../cyclopeptides/">Cyclopeptide Finder</a>
                  </li>
              </ul>
              <p class="caption"><span class="caption-text">Local Installation</span></p>
              <ul>
                  <li class="toctree-l1"><a class="reference internal" href="../../install/">Installation</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../../command_line/">Starting command line runs</a>
                  </li>
              </ul>
              <p class="caption"><span class="caption-text">Change Log</span></p>
              <ul>
                  <li class="toctree-l1"><a class="reference internal" href="../../changelog/">Overview</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../../changelog/2.0/">Version 2.0</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../../changelog/1.0/">Version 1.0</a>
                  </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../../faq/">FAQ</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../../glossary/">Glossary</a>
                </li>
              </ul>
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="../../about/">About</a>
                </li>
              </ul>
      </div>
    </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">
      <nav class="wy-nav-top" role="navigation" aria-label="Mobile navigation menu">
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../..">plantiSMASH Documentation</a>
        
      </nav>
      <div class="wy-nav-content">
        <div class="rst-content"><div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href="../.." class="icon icon-home" aria-label="Docs"></a></li>
          <li class="breadcrumb-item">Generic modules</li>
      <li class="breadcrumb-item active">ClusterBLAST</li>
    <li class="wy-breadcrumbs-aside">
    </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
            <div class="section" itemprop="articleBody">
              
                <h1 id="clusterblast">ClusterBLAST<a class="headerlink" href="#clusterblast" title="Permanent link">#</a></h1>
<p>If you enabled Gene Cluster Comparative Analysis (ClusterBlast), this panel will display the top ten plant gene clusters from the <code>plantiSMASH ClusterBLAST database</code> that are most similar to a detected gene cluster, visually aligned to it. The drop-down selection menu can be used to browse through the gene clusters. Genes with the same colour are putative homologs based on significant Blast (Diamond) hits between them.</p>
<p>The <code>plantiSMASH ClusterBLAST database</code> is defined with a minimum of 2 domains needed from the BGC rules. </p>
<p><img alt="ClusterBLAST output thalianol" src="../../assets/images/clusterblast_output.png" /></p>
<p>Genes with matching colours are interrelated.
Percentage identity values per hit are shown when clicking the gene in question,
with a minimum percentage identity between genes of 30%.</p>
<p>It is normal to have multiple genes hitting for some types of genes. </p>
<h3 id="clusterblast-database">ClusterBLAST database<a class="headerlink" href="#clusterblast-database" title="Permanent link">#</a></h3>
<p>More information about how to update the clusterBLAST database are available in the <a href="https://github.com/plantismash/plantismash/wiki">plantiSMASH Wiki</a>. </p>
<h3 id="ranking-system">Ranking system<a class="headerlink" href="#ranking-system" title="Permanent link">#</a></h3>
<p>Reference areas are sorted first based on an empirical similarity score <code>S = h + H + s + S + B</code>, in which:
- <code>h</code> is the number of query genes with a significant hit
- <code>H</code> is the number of core query genes with a significant hit
- <code>s</code> is the number of gene pairs with conserved synteny
- <code>S</code> is the number of gene pairs with conserved synteny involving a core gene
- and <code>B</code> is a core gene bonus</p>
<p>If the similarity scores are equal, the hits are subsequently ranked based on
the cumulative BlastP bit scores between the gene clusters.</p>
<h3 id="similarity-percentage">Similarity percentage<a class="headerlink" href="#similarity-percentage" title="Permanent link">#</a></h3>
<p>Similarity in the description, e.g. <code>87% of genes show similarity</code>,
is the percentage of genes within the reference that have a hit to any genes in the query.</p>
<p>As gene hits are not required to be 100% identity and query genes may hit multiple reference genes,
this total similarity percentage is no guarantee that the region is exactly the same.
In the case of KnownClusterBlast, this also means that there is no guarantee that the compound(s) recorded for that MIBiG entry will be produce by the region.</p>
<p>Even if 100% of genes have a hit for a reference, it may be less relevant than a lower similarity.
Some cluster types, e.g. NRPS clusters, may only need a few aminos changed in gene translations to have a completely different product.</p>
<p>In all cases, manual verification is required before assuming that the region produces the same compound as the reference.</p>
<h4 id="example-1-low-similarity-good-match">Example 1: low similarity, good match<a class="headerlink" href="#example-1-low-similarity-good-match" title="Permanent link">#</a></h4>
<p>Reference area <code>R</code> has 70% of genes showing similarity to the query region <code>Q</code>.
All genes with hits are very high identity in their hits, at 95% or higher.</p>
<p>The missing genes from <code>R</code> are all at one end of the reference area.
These genes are also present in the genome antiSMASH was run on,
but are outside <code>Q</code> due to the size of <code>R</code> being exceptionally large.</p>
<p>After manually checking these extra genes and seeing that they're similar to the missing genes,
it's much, much more likely that the genome matches the reference.</p>
<h4 id="example-2-perfect-similarity-poor-match">Example 2: perfect similarity, poor match<a class="headerlink" href="#example-2-perfect-similarity-poor-match" title="Permanent link">#</a></h4>
<p>Reference area <code>R</code> has 100% of genes showing similarity to the query region <code>Q</code>.
None of the genes have a percentage identity in individual hits greater than 60%.</p>
<p>While it is still possible that <code>Q</code> produces the same compound as <code>R</code>,
it will depend a great deal on the type of cluster and exactly which parts of the genes are similar.</p>
<h4 id="example-3-high-similarity-poor-match">Example 3: high similarity, poor match<a class="headerlink" href="#example-3-high-similarity-poor-match" title="Permanent link">#</a></h4>
<p>Reference area <code>R</code> has very high (but not 100%) similarity, with all but one gene in <code>R</code> having similarity to genes in the query region <code>Q</code>.
All of the matching genes have very high identity in their hits.</p>
<p>The gene in <code>R</code> that is missing in <code>Q</code> should be in the middle of <code>Q</code> and is a core gene responsible for the scaffold of the compound.</p>
<p>While it is still possible that <code>Q</code> produces the same compound as <code>R</code>, due to the functionality being present elsewhere, but it is much less likely.</p>
              
            </div>
          </div>
<hr>
<p>
    
    <small>© 2025 <a href="https://github.com/elenadelpup" target="_blank">Elena Del Pup</a><br></small>
    

    
    <small><br><i>Last updated 2025-10-20</i></small>
    
</p>

        </div>
      </div>

    </section>

  </div>

  <div class="rst-versions" role="note" aria-label="Versions">
  <span class="rst-current-version" data-toggle="rst-current-version">
    
    
      <span><a href="../clusterrules/" style="color: #fcfcfc">&laquo; Previous</a></span>
    
    
      <span><a href="../knownclusterblast/" style="color: #fcfcfc">Next &raquo;</a></span>
    
  </span>
</div>
    <script src="../../js/jquery-3.6.0.min.js"></script>
    <script>var base_url = "../..";</script>
    <script src="../../js/theme_extra.js"></script>
    <script src="../../js/theme.js"></script>
      <script src="../../search/main.js"></script>
    <script>
        jQuery(function () {
            SphinxRtdTheme.Navigation.enable(true);
        });
    </script>

</body>
</html>
