# About plantiSMASH 

plantiSMASH allows the rapid genome-wide identification, annotation, and analysis of specialized metabolite biosynthetic gene clusters (BGCs) across the plant kingdom. It is a specialized extension of the widely used antiSMASH tool, tailored specifically to target plant genomes.

plantiSMASH is powered by several open-source tools:
NCBI BLAST+
Diamond
HMMer 3
Muscle 3
GlimmerHMM
CD-HIT
PySVG
JQuery SVG
JQuery DataTables
InCHlib
vis.js
pplacer
GraPhlAn

# antiSMASH contributors 

plantiSMASH version 2.0 is the product of a collaborative effort between:

Bioinformatics Group, Wageningen University & Research
Department of Metabolic Biology, John Innes Centre, Norwich
The Robert H. Smith Institute of Plant Sciences and Genetics in Agriculture, Hebrew University of Jerusalem
DOE Joint Genome Institute, Lawrence Berkeley National Labs
Supported by the Graduate School for Experimental Plant Sciences (EPS) and Vidi Grant VI.Vidi.213.183 from The Netherlands Organization for Scientific Research (NWO).

# How to cite 

If you have found plantiSMASH useful, please cite:

plantiSMASH: automated identification, annotation and expression analysis of plant biosynthetic gene clusters.
Satria A. Kautsar, Hernando G Suarez Duran, Kai Blin, Anne Osbourn & Marnix H. Medema
Nucleic Acids Research, 45(W1), W55-W63. (2017) [https://doi.org/10.1093/nar/gkx305](https://doi.org/10.1093/nar/gkx305)