plantiSMASH Documentation
--------------------------


Welcome to the documentation of **plantiSMASH**. 

### How to Use plantiSMASH - Public Web Version

The easiest way to get started with plantiSMASH is to use [the public webserver](https://plantismash.bioinformatics.nl/).

The [Glossary](glossary.md) explains some of the abbreviations used in plantiSMASH and also provides an overview of the biosynthetic classes that plantiSMASH can detect.

Frequently occurring questions can be found in the [FAQ](faq.md) section.
Please consult the FAQs before raising an [Issue](https://github.com/plantismash/plantismash/issues) on the antiSMASH GitHub page.

For more details on how to use plantiSMASH, see our Springer's Protocols book chapter:
   
[Kautsar, S. A., Duran, H. G. S., & Medema, M. H. (2018).
 *Genomic identification and analysis of specialized metabolite biosynthetic gene clusters in plants using PlantiSMASH.* 
 In Plant Chemical Genomics (pp. 173-188). Humana Press, New York, NY.](https://link.springer.com/protocol/10.1007/978-1-4939-7874-8_15) 


### How to Use plantiSMASH - Local Installation

If you need to run many analyses or custom analyses, please download and install a local copy of antiSMASH.
Running large-scale analyses locally instead of on the webservice reduces the load and helps us to keep plantiSMASH available for everybody.

For instructions on how to install antiSMASH locally, please see the [Install Guide](install.md).

### For developers 

This documentation is mainly addressed to users wanting to know more about the individual analyses of plantiSMASH.
If you are interested in contributing to antiSMASH or in the technical design, please see the [plantiSMASH GitHub Wiki](https://github.com/plantismash/plantismash/wiki).


### antiSMASH 

If you want to know more about the antiSMASH framework, check the [antiSMASH Documentation](https://docs.antismash.secondarymetabolites.org/). 